activesupport/lib/active_support/duration.rb

```diff
@@ -130,6 +133,23 @@ def respond_to_missing?(method, include_private=false) #:nodoc:
       @value.respond_to?(method, include_private)
     end
 
+    # Creates a new Duration from string formatted according to ISO 8601 Duration.
+    #
+    # See [ISO 8601](http://en.wikipedia.org/wiki/ISO_8601#Durations) for more information.
```

Comment:
`See {ISO 8601}[http://en.wikipedia.org/wiki/ISO_8601#Durations] for more information.`